docs/specs/openapi.yaml

```diff
@@ -153,6 +79,314 @@ paths:
+        - changes
+      description: Fetch information about a Change given its ID.
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: ID of the change
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetChange'
+              examples:
+                get change:
+                  value:
+                    type: sync
+                    status-code: 200
+                    status: OK
+                    result:
+                      id: 72
+                      kind: autostart
+                      summary: "Autostart service \"svc1\""
+                      status: Done
+                      tasks: 
+                        - id: 72
+                          kind: start
+                          summary: "Start service \"svc1\""
+                          status: Done,
+                          progress:
+                            label: ""
+                            done: 1
+                            total: 1
+                          spawn-time: "2024-12-22T20:05:24.855436063+08:00"
+                          ready-time: "2024-12-22T20:05:25.866100522+08:00"
+                      ready: true,
+                      spawn-time: "2024-12-22T20:05:24.855443855+08:00"
+                      ready-time: "2024-12-22T20:05:25.866103439+08:00"
+    post:
+      summary: Perform an action on a change.
+      tags:
+        - changes
+      description: Perform an action on a change. Currently the only supported action is "abort".
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: ID of the change
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                action:
+                  type: string
+                  description: The action to perform on the change.
+                  enum: [abort]
+      responses:
+        '200':
+          description: abort change
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetChange'
+              examples:
+                abort change:
+                  value:
+                    type: sync
+                    status-code: 200
+                    status: OK
+                    result:
+                      id: 76
+                      kind: perform-check
+                      summary: "Perform HTTP check \"svc1-up\""
+                      status: Abort
+                      tasks: 
+                        - id: 76
+                          kind: perform-check
+                          summary: "Perform HTTP check \"svc1-up\""
```

Comment:
Same question about whether the change ID & summary would be the same as the task ID & summary

Reply:
Example updated to a real-world example with different IDs and also updated "examples" to a single example in JSON format.